schema/swagger.yml

```diff
@@ -595,6 +595,46 @@ paths:
           schema:
             $ref: '#/definitions/UnexpectedError'
 
+  /delegates/{publicKey}/forged:
+    x-swagger-router-controller: delegates
+    get:
+      tags:
+      - Delegates
+      summary: Requests forging statistics by delegate publicKey
+      operationId: getForgedByAccount
+      x-lisk-cache-enabled: true
+      description:
+        By passing an existing delegate publicKey, you can read
+        their forging statistics within a window of time
+      produces:
+      - application/json
+      parameters:
+        - in: path
+          name: publicKey
+          description: Public key to query
+          required: true
+          type: string
+          format: publicKey
+        - $ref: '#/parameters/fromTimestamp'
+        - $ref: '#/parameters/toTimestamp'
```

Comment:
You are using some default values for these parameters. 

```
fromTimestamp: constants.epochTime.getTime(),
toTimestamp: Date.now(),
```
Although we can't assign these as default values to these params. But we should mentioned it into the API endpoint description.